### PR TITLE
fix: getkubernetesResourceManagers popup

### PR DESCRIPTION
**Pull request by amandavialva01**
## Description
Fix `getKubernetesResourceManager` toast so that it doesn't show up on login screen after a user logs out.

## Test Plan
CI passes.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/components/WorkspaceCreateModal.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ const isNonK8RMError = (e: unknown): boolean => {
 };
 
 const isNotAuthorizedErr = (e: unknown): boolean => {
-  return e instanceof DetError && e.sourceErr instanceof Response && e.sourceErr['status'] === 403;
+  return e instanceof DetError && e.sourceErr instanceof Response && (e.sourceErr['status'] === 403 || e.sourceErr['status'] === 401);
```
Comment (hkang1) [inline webui/react/src/components/WorkspaceCreateModal.tsx:64]: **question:**
- is there enough overlap of this and `isAuthFailure`?
- not for this PR but maybe this function can be made reusable into `src/utils/services.ts` if this is deemed common enough of a situation?

File: webui/react/src/components/WorkspaceCreateModal.tsx [hunk 2]
```diff
@@ -110,7 +110,7 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
           handleError(e, {
             level: ErrorLevel.Error,
             publicMessage: 'Failed to fetch list of workspace namespace bindings.',
-            silent: false,
+            silent: true,
```
Comment (hkang1) [inline webui/react/src/components/WorkspaceCreateModal.tsx:113]: **question:** this effectively silences this error for good from the user perspective but will still get logged, confirming that it is what we want (as opposed to conditionally showing it to the user)

**Review by hkang1 — COMMENTED** [review]
It's been a while since I've been in this code base, but had some questions on the changes made :pray:

**Comment by salonig23** [comment]
> is this still necessary with #9735?

No it is not. @amandavialva01 can you close this PR?

**Comment by amandavialva01** [comment]
> > is this still necessary with #9735?
> 
> No it is not. @amandavialva01 can you close this PR?

Yea I can close this, is there a PR where the issue of the `getKubernetesResourceManagers` toast gets resolved that I can link in the closing comment @salonig23?

File: webui/react/src/components/WorkspaceCreateModal.tsx (hunk 2 above)
Comment (amandavialva01) [reply webui/react/src/components/WorkspaceCreateModal.tsx:113]: cc @salonig23

**Pull request closed without merging**